net_util/src/open_tap.rs

```diff
@@ -85,10 +85,23 @@ pub fn open_tap(
     for i in 0..num_rx_q {
         let tap: Tap;
         if i == 0 {
-            tap = match if_name {
-                Some(name) => Tap::open_named(name, num_rx_q, flags).map_err(Error::TapOpen)?,
-                None => Tap::new(num_rx_q).map_err(Error::TapOpen)?,
+            let created: bool;
```

Comment:
looks more verbose than it actually is: just added `let created: bool;` and the `match{}` now returns a `(Tap, bool)` tuple